src/components/Entry/Summary/OverlappingEntries/index.tsx

```diff
@@ -33,11 +33,19 @@ const OverlappingEntries = ({ metadata }: { metadata: EntryMetadata }) => {
     );
 
   return (
-    <div>
-      <h4>
-        {metadata.type === 'homologous_superfamily'
-          ? 'Overlapping entries'
-          : 'Overlapping homologous superfamilies'}
+    <tr>
+      <td>
+        {metadata.type === 'homologous_superfamily' ? (
+          'Overlapping entries'
+        ) : (
+          <>
+            Overlapping
+            <br />
+            homologous
+            <br />
+            superfamilies
```

Comment:
Living with the consequences of my own decisions!

I put a css rule so those titles don't wrap in multiple lines, because I think it doesn't look nice when the title is split.
But this particular title is way too long 🤦🏽‍♂️

I know looks ugly, but thought that been a single case it wont matter, and nobody is going to notice